src/commonMain/kotlin/com/epam/drill/request/Plugins.kt

```diff
@@ -0,0 +1,5 @@
+package com.epam.drill.request
+
+expect object Plugins {
+    fun fillProbes()
```

Comment:
agent doesnt know anything about Probes... what is it? this method should **PROCESS INCOMING NETWORK REQUEST** it shouldn do anything with Probes...